app/Commands/MigrateCommand.php

```diff
@@ -822,7 +824,7 @@ private function replaceDatabaseURLsLocal()
         $targetSiteURL = "hale.docker";
 
         if ($this->blogID !== null) {
-            $urlFlag = "--url=$targetSiteURL";
+            $urlFlag = "--url=$domainPath 'wp_{$this->blogID}_*' --all-tables-with-prefix";
```

Comment:
domainPath is not set

Reply:
Actually $domainPath should be changed to $targetSiteURL in this instance. I'll change. Although I made this change here, all of the local code I was going to review as we haven't really looked at it for a while and I can't remember how compatible it is.

Reply:
Reverted back.